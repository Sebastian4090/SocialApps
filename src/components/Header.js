import React from 'react';

const Header = React.memo((props) => {
    console.log('header')
    return <h1 className='f1'>SocialApps</h1>
})

export default Header;