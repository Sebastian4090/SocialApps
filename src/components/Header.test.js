import { shallow } from 'enzyme';
import React from 'react';
import Header from './Header';

it('expect to render Header component', () => {
    expect(shallow(<Header />)).toMatchSnapshot()
})