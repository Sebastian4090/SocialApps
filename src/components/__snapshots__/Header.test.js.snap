// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Header component 1`] = `
<h1
  className="f1"
>
  SocialApps
</h1>
`;
